recipes/or-tools/build.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+cmake ${CMAKE_ARGS} -DCMAKE_INSTALL_PREFIX=$PREFIX \
+      -DCMAKE_BUILD_TYPE=Release \
+      -DCMAKE_INSTALL_LIBDIR=lib \
+      -BUILD_DEPS=OFF \
+      -DUSE_SCIP=OFF \
+      $SRC_DIR
```

Comment:
what is `$SRC_DIR` ? for CMake the idiomatic way is
`-S$SRC_DIR -B$BUILD_DIR`

then building using
`cmake --build $BUILD_DIR`

Reply:
The newest cmake conda-forge has is 3.21.2 so let me just restrict it to >=3.13 and then use the newer tools.

Reply:
EDIT: or-tools depends on 3.15 (and soon 3.18 when building the python part)
ref: